update_data.bash

```diff
@@ -16,22 +17,28 @@ fi
 function metadata_url {
 	local os=$1
 	local arch=$2
+	local release=$3
 
-	echo "https://joschi.github.io/java-metadata/metadata/ga/${os}/${arch}.json"
+	echo "https://joschi.github.io/java-metadata/metadata/${release}/${os}/${arch}.json"
 }
 
 function fetch_metadata {
 	local os=$1
 	local arch=$2
-	local url
-	url=$(metadata_url "$os" "$arch")
 
 	local args=('-s' '-f' '--compressed' '-H' "Accept: application/json")
 	if [[ -n "${GITHUB_API_TOKEN:-}" ]]; then
 		args+=('-H' "Authorization: token $GITHUB_API_TOKEN")
 	fi
 
-	curl "${args[@]}" -o "${DATA_DIR}/jdk-${os}-${arch}.json" "${url}"
+	for RELEASE_TYPE in $LIST_RELEASE_TYPE
```

Comment:
Why not iterate over `LIST_RELEASE_TYPE` in line 37-43, similar to `LIST_OS` and `LIST_ARCH`?

Reply:
I did that first but I moved it into fetch because merging the ea and ga files look different previously. Now there is no reason keeping the loop there, I fixed it.